pkgs/top-level/python-packages.nix

```
@@ -3278,6 +3278,10 @@ in {
 
   kitchen = callPackage ../development/python-modules/kitchen { };
 
+  kivy = callPackage ../development/python-modules/kivy { };
```

to avoid mesa from python scope
```suggestion
  kivy = callPackage ../development/python-modules/kivy {
    inherit (pkgs) mesa ... ;
  };
```

---

Yep indeed. Fixed.